client/src/components/share/index.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { ShareTemplate } from './share-template';
+import { useShare } from './use-share';
+
+interface ShareProps {
+  superBlock: string;
+  block: string;
+}
+
+export const Share: React.FC<ShareProps> = ({ superBlock, block }) => {
```

Comment:
This isn't Functional component, as there isn't `Share.something`, and using it has its own downside, you can find more here https://github.com/facebook/create-react-app/pull/8177

```suggestion
export const Share = ({ superBlock, block }: ShareProps): JSX.Element => {
```

Reply:
Updating with this change